lib/cc/services/github_pull_requests.rb

```diff
@@ -104,6 +107,9 @@ def update_status(state, description)
         target_url:  @payload["details_url"],
         context:     "codeclimate"
       }
+      if state == "error"
+        params.delete(:target_url)
```

Comment:
At first glance, I don't love this. Why not just not include the target_url in the original payload. I don't think this class should be responsible for "overriding" the payload in this way. /c @gordondiggs @abaldwinhubter @pbrisbin